src/components/common/Problem.vue

```diff
@@ -29,34 +30,107 @@ export default defineComponent({
 		const _file: Ref<string> = ref(props.file);
 		const renderDiv = ref<HTMLElement>();
 
+		// modified from https://github.com/tserkov/vue-plugin-load-script
+		async function loadScript(src: string) {
+			console.log('loading scripts...', src);
+			// eslint-disable-line no-param-reassign
+			return new Promise<void>(function (resolve, reject) {
+				let shouldAppend = false;
+				let el: HTMLScriptElement | HTMLLinkElement | null;
+				create_el: if (/\.js(?:\??[0-9a-zA-Z=]*)$/.exec(src)) {
+					el = document.querySelector('script[src="' + src + '"]');
+					if (el && el.hasAttribute('data-loaded')) {
+						resolve();
+						console.log(`${src} seems to have already been loaded!`);
+						return;
+					} else if (el) {
+						break create_el;
+					}
+
+					el = document.createElement('script');
+					el.type = 'text/javascript';
+					el.async = true;
```

Comment:
This should be changed to `el.defer = true`.  Asynchronous loading does not guarantee order of loading, and defer does.

Reply:
I should add that I doubt defer will actually have any effect in this situation since these scripts are being dynamically injected after the DOM has already loaded in any case.  On the other hand async will have the negative implications I mentioned.